backend/src/bork-protocol/utils/token-monitor/filter-tokens-by-interest.ts

```diff
@@ -0,0 +1,225 @@
+import type { TweetQueueService } from '@/services/twitter/analysis-queue.service';
+import type { TwitterConfigService } from '@/services/twitter/twitter-config-service';
+import type { TwitterService } from '@/services/twitter/twitter-service';
+import type {
+  EnrichedToken,
+  InterestingToken,
+  TokenProfile,
+} from '@/types/token-monitor/token';
+import { elizaLogger } from '@elizaos/core';
+import { searchTokenTweets } from './token-search';
+
+/**
+ * Clears old interesting tokens to prevent stale data
+ * @param interestingTokens Map of interesting tokens
+ * @param maxAgeHours Maximum age of tokens to keep (in hours)
+ * @returns Updated map of interesting tokens
+ */
+export function clearOldInterestingTokens(
```

Comment:
so right now we just use new coins? basically get in before initial pump at this point?
Not that I think we should do differently, but if that's the case our agent should be geared towards that